packages/tasit-act/src/ethers.test.js

```diff
@@ -0,0 +1,92 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
+// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
+const contractAddress = "0x6C4A015797DDDd87866451914eCe1e8b19261931";
+
+let provider, wallet, contract;
```

Comment:
Same comment as for the tasit-account tests

Reply:
Same `beforeEach` doubt.

Reply:
I'll close this conversation and we can discuss there https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243321392